packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -954,6 +954,10 @@ mixin WidgetInspectorService {
     _errorsSinceReload = 0;
   }
 
+  /// Whether structured errors are enabled.
+  ///
+  /// Structured errors provide semantic information that can be used by IDEs
+  /// to enhance the display of errors with rich formatting.
```

Comment:
would be good to point to things like FlutterError and so on where we have more detail